pyscriptjs/src/python/pyscript.py

```diff
@@ -124,6 +124,21 @@ def write(element_id, value, append=False, exec_id=0):
         )
 
 
+current_py_script_tag = None
```

Comment:
UPPERCASE ;)
Or, even better: instead of using a global, you can store the value as an attribue of the function object itself:
```python
def display():
    ...
display._default_target_id = None

def set_default_display_target_id(id):
    display._default_target_id = id
```

Reply:
oops
true about the uppercase
alright 👍 I did it like this cause you said global was the best solution? but you actually just mean we shouldn't use an object?
Also, I don't understand why it's a problem to use an object. I think it's possible to just pass it on the `base.ts`, on the same place where I can the function. What do you think?

Reply:
> I did it like this cause you said global was the best solution? but you actually just mean we shouldn't use an object? 

yes exactly. What I meant was "using global state is simpler than injecting a per-script object".
That said, you can set global state in two ways:
1. with a "real" global variable, as you did
2. as an attribute of the global `display` function

The difference is shallow, but usually in Python (2) is preferred because it makes it clear that the global is specific to this function

> Also, I don't understand why it's a problem to use an object. I think it's possible to just pass it on the `base.ts`, on the same place where I call the `set_current_display_target` function. What do you think?

feel free to try and tell me how it goes :)